### DISCUSSION TITLE
PR Discussion [Jul 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2846] [Discord Threads]
> 🙋‍♂️ **moi**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### which PR u've submitted? PR2658?  
#  
> 🙋‍♂️ **moi**  
#### yes exactly  
#  
> 🙋‍♂️ **moi**  
#### Sorry I just saw your message, I was trying to solve the datetime issue that wing kwong highlighted  
#  
> 🙋‍♂️ **wkw**  
#### my another colleague s reviewing as well. we can have a detailed discussion after that.  
#  
> 🙋‍♂️ **moi**  
#### Is there a documentation to build app or plugin for nocodb ? I did not really find one on your documentation.  
#  
> 🙋‍♂️ **moi**  
#### Maybe it would be an easier process for me to work and improve nocodb 🙂  
#  
> 🙋‍♂️ **moi**  
#### Any news ?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/996690146541977630)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
